src/app/newSettings/Sections/Workspace/Members/containers/MemberDetailsContainer.tsx

```diff
@@ -77,7 +100,7 @@ const MemberDetailsContainer = ({ member }: MemberDetailsContainer) => {
       view: <TeamsTab role={memberRole} teams={teams.teams} isTeams={teams.isTeams} />,
     },
   ];
-  const [activeTab, setActiveTab] = useState<ActiveTab>(tabs[0]);
+  // const [activeTab, setActiveTab] = useState<ActiveTab>(tabs[0]);
```

Comment:
Leave a comment if this is gonna be used after MVP

Reply:
Done